File: Architecture.Web/ClientApp/src/app/Shared/Entity/Dashboard/Offer-Info.ts
import { Auditable } from '../Common/auditable';
import { JobInfo } from '../Users/JobInfo';

export class OfferInfo extends Auditable {
    public offerInfoId: number;
    public jobId: number;
    public jobInfo: JobInfo;
    public jobTitle: string;
    public profileId: number;
    public profileName: string;
    public acceptedOperatorId: string;
    public operatorAcceptedDate: Date;
    public validatorId: string;
    public validatorName: string;
    public validationDate: Date;
    public offerStatusId: number;
    public status: string; 
    public currentUserId: string;
    public createdDate: string;
}

File: Architecture.Core/Entities/OfferInfo.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;
using System;
using System.Collections.Generic;
using System.Text;

namespace Architecture.Core.Entities
{
    public class OfferInfo : Auditable
    {
        public int OfferInfoId { get; set; }       
        public int JobId { get; set; }
        public virtual JobInfo JobInfo { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public string AcceptedOperatorId { get; set; }
        public DateTime? OperatorAcceptedDate { get; set; }
        public string ValidatorId { get; set; }
        public DateTime? ValidationDate { get; set; }
        public int? OfferStatusId { get; set; }
        public virtual OfferStatus OfferStatus { get;set;}
        public string Status { get; set; }
        public Guid? CurrentUserId { get; set; }
        public string DocumentSrc { get; set; }
        public string Code { get { return $"NC-{this.OfferInfoId.ToString("D4")}"; } }

    }
}
